initial_setup/common.py

```diff
@@ -52,3 +52,54 @@ def collectCategoriesAndSpokes(hub_instance, spoke_parent_class):
         ret[spoke.category].append(spoke)
 
     return ret
+
+def parse_os_release_lines(osreleaselines):
+    """man os-release for format
+
+       :return: dictionary mapping of line
+       :rtype: dictionary
+    """
+    osrel = {}
+
+    for line in osreleaselines:
+        if line.endswith("\n"):
+            line = line[:-1]
+
+        kvp = line.split('=')
+        if len(kvp) < 2:
+            continue
+
+        key = kvp[0]
+        value = kvp[1]
+        if len(kvp) > 2:
+            value = "=".join(kvp[1:])
```

Comment:
This is not necessary if ``line.partition('=')`` is used (``line.split('=', 1)`` would be enough too).